TcpIpInput.cpp

```diff
@@ -33,30 +33,47 @@ TcpIpInput::TcpIpInput(uint16_t port, unsigned short domain, unsigned int type){
         perror("bind failed");
         exit(EXIT_FAILURE);
     }
+
 }
 
 std::string TcpIpInput::getData(){
     int newSocket, valRead;
     int addrLen = sizeof(this->mAddress);
+    char const *hello = "Hello from server";
     if (listen(this->mServerFile, 3) < 0)
     {
         perror("listen");
         exit(EXIT_FAILURE);
     }
+
     if ((newSocket = accept(this->mServerFile, (struct sockaddr *)&(this->mAddress),
                        (socklen_t*)&addrLen))<0)
     {
         perror("accept");
         exit(EXIT_FAILURE);
     }
     valRead = read(newSocket , this->mBuffer, MAX_BUFFER_SIZE);
-    printf("%s\n",this->mBuffer );
+    send(newSocket ,hello,strlen(hello),0);
```

Comment:
I don't understand. Why do you send something on the socket instead of just receiving data? I doesn't fix the problem, this is only some weird workaround.